testsuite/features/step_definitions/setup_steps.rb

```diff
@@ -505,3 +505,31 @@
     step %(I check "#{unique_repo_name}" in the list)
   end
 end
+
+When(/^I prepare the development repositories of "([^"]*)" as part of "([^"]*)" channel$/) do |host, channel_label|
+  target = get_target(host)
+  repo_urls =
+    case target.os_family
+    when 'debian', 'ubuntu'
```

Comment:
use `deb_host()?` from commonlib ?

Reply:
I don't like the implementation of deb_host, I don't like to depend on the hostname.
But I can refactor that method, to use my approach.

Reply:
ok